labelbox/schema/model_run.py

```diff
@@ -286,17 +286,17 @@ def add_predictions(
         Returns:
```

Comment:
Since MEAPredictionImport tests uses this codepath for testing, we don't need additional unit tests for MEAPredictionImport; updating this method to use the new create method provides coverage with existing unit tests.